code/modules/clothing/under/color.dm

```diff
@@ -103,6 +103,55 @@
 	item_color = "yellow"
 	flags = ONESIZEFITSALL
 
+
+/obj/item/clothing/under/color/yellow/parole
+	name = "parole jumpsuit"
+	desc = ""

```

Comment:
Better to just have the unlocked description by default, then have the description change when it's impacted by various conditions.

and use `initial(desc)` when resetting it back.